js2-imenu-extras.el

```diff
@@ -221,6 +229,161 @@ Currently used for jQuery widgets, Dojo and Enyo declarations."
                                      (list name-value))
                                    (js2-node-abs-pos methods))))))
 
+(defun js2-imenu-record-mocha-describe ()
+  "Populate `js2-imenu-recorder' with mocha-like describe/it/beforeEach/… nodes."
+  (let ((node (js2-node-at-point (1- (point)))))
+    (when (js2-imenu-extras--mocha-top-level-describe-p node)
+      (js2-imenu-extras--mocha-visit-node node (list)))))
```

Comment:
FWIW, `(list)` evaluates to `nil`.

Reply:
IMO, `(list)` better conveys the meaning of the parameter being a list than `nil`. I'm fine changing it to `nil` if you want me to.

Reply:
It's all right.